tools/pot/README.md

```diff
@@ -20,50 +20,72 @@
+* A representative calibration dataset representing a use case scenario, e.g. 300 images. 
+
+Figure below shows the optimization workflow:
+![](docs/images/workflow_simple.png) 
 
 Post-training Optimization Tool provides the following key
 features:
 
 * Two post-training 8-bit quantization algorithms: fast [DefaultQuantization](openvino/tools/pot/algorithms/quantization/default/README.md) and precise [AccuracyAwareQuantization](openvino/tools/pot/algorithms/quantization/accuracy_aware/README.md).
 * Compression for different hardware targets such as CPU and GPU.
 * Multiple domains: Computer Vision, Natural Language Processing, Recommendation Systems, Speech Recognition.
+* [Command-line tool](docs/CLI.md) that provides a simple interface for basic use cases.
 * [API](openvino/tools/pot/api/README.md) that helps to apply optimization methods within a custom inference script written with OpenVINO Python* API.
-* Symmetric and asymmetric quantization schemes. For details, see the [Quantization](openvino/tools/pot/algorithms/quantization/README.md) section.
-* Per-channel quantization for Convolutional and Fully-Connected layers.
+* (Experimental) [Ranger algorithm](@ref pot_ranger_README) for model prodection in safity critical cases.
 
 The tool is aimed to fully automate the model transformation process without a need to change the model on the user's side. For details about 
 the low-precision flow in OpenVINO&trade;, see the [Low Precision Optimization Guide](docs/LowPrecisionOptimizationGuide.md).
 
 For benchmarking results collected for the models optimized with POT tool, see [INT8 vs FP32 Comparison on Select Networks and Platforms](@ref openvino_docs_performance_int8_vs_fp32).
 
-POT is opensourced on GitHub as a part of [https://github.com/openvinotoolkit/openvino](https://github.com/openvinotoolkit/openvino).
+POT is opensourced on GitHub as a part of OpenVINO and available at https://github.com/openvinotoolkit/openvino/tools/pot.
 
 Further documentation presumes that you are familiar with the basic Deep Learning concepts, such as model inference,
 dataset preparation, model optimization, as well as with the OpenVINO&trade; toolkit and its components such 
 as  [Model Optimizer](@ref openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide) 
 and [Accuracy Checker Tool](@ref omz_tools_accuracy_checker).
 
-## Use POT
-![](docs/images/workflow.png) 
+## Usage options
+![](docs/images/use_cases.png) 
+
+The POT provides three basic usage options:
+* **Command-line interface (CLI)**:
+  * [**Simplified mode**](@ref pot_docs_simplified_mode): this option can be used if the model from Computer Vision domain and there is an unannotated dataset that can be used for optimization. Please note that there can be deviation of accuracy after optimization using this method.
+  * [**Model Zoo flow**](@ref pot_compression_cli_README): this option is recommended if the model from OpenVINO&trade; 
```

Comment:
quantizer.py has a limited support of quantizable models but the link to this script makes sense. AC is a part of OMZ, I don't see any confusion since all the available AC configs are located in OMZ.

Reply:
Why do you want to refer to Model Zoo instead of simply saying about AC support? Any OMZ model has AC support so it already includes "if the model is imported from OpenVINO™ [Model Zoo](https://github.com/openvinotoolkit/open_model_zoo) or".
Anyway we always say "through AC" or "with AC support" so I believe "Model Zoo" name is confusing here. It's even misleading because the user might think that it's only for OMZ models.

Reply:
>>Why do you want to refer to Model Zoo instead of simply saying about AC support?
Because this document and this part for newcomers who don't know what is AccuracyChecker. Model Zoo at least says something.

Reply:
Here we just would like to highlight what we have for newcomers. Details and particular examples can be found in the corresponding docs:
- Dedicated document/feature spec - all the options and litations
- Example - ready-to-use code snippets

Reply:
I think you don't see my point: I believe we should guide the user, but you want the user to figure out what to do next by reading all documentation. Let's be honest, users are lazy and they will not read a lot of documentation, they will either stop or try something what is obvious. I don't propose describing all details on the main page but it would be useful if we say: in this case go this way, in that case - that way.

Reply:
Let's see what @tsavina can propose here.